hydrus/client/ClientImageHandling.py

```diff
@@ -61,17 +61,13 @@ def GenerateShapePerceptualHashes( path, mime ):
         
         numpy_alpha_float = numpy_alpha / 255.0
         
-        numpy_image_bgr = numpy_image[ :, :, :3 ]
+        numpy_alpha_inverted = 255.0 - numpy_alpha
```

Comment:
```py
numpy_image_bgr = numpy_image[ :, :, :3 ]
```
Yields the image less the alpha channel the modified code seems to skip this step, I'm not sure if the intermediate steps take care of that make sure to test the output shape if the input had transparency.

Reply:
I didnt chanmge this at first, but after some testing, including the alpha channel in the conversion seems to have no effect on the output. (as you can see above)
The only thing thats different between the two is that one output has a completely white alpha channel, while the other one doesnt have one at all.
This resulted in a 400 byte bigger image on the new modified code